docs/catalogue/cat_data-manager.md

```diff
@@ -0,0 +1,259 @@
+which can be used as a basis for integrated analysis. In some projects, data may be made available via [DataSHIELD](https://www.datashield.org/).
+In these cases each resource stores the data locally in a [MOLGENIS Armadillo](../armadillo/) DataSHIELD server.
+
+This page describes the process of filling out metadata and variable information for a resource
+and uploading this to the catalogue. Users of this guide are expected to be data managers within the organisations
+with access to the resources to be included in the catalogue, who can submit metadata, source variables
+(also called collected variables) and variable mappings. In addition, it can be used by central data managers
+of networks with a common data model such as LifeCycle or LongITools, who can submit metadata and target variables
+(also called the harmonised variables or common data model).
+This document assumes you have received login details for upload of your variables and metadata.
+If this is not the case, see the section on [requesting access](#request-access).
+
+## Staging areas
+
+The metadata of the resource are first uploaded into what are called the 'staging areas' of the catalogue.
+Later on the metadata are transferred to the main catalogue
+Use of a staging area allows for review before the information becomes available through the live catalogue.
+If the data in your staging area is ready for transfer you can let us know by emailing us through [MOLGENIS support](mailto:support@molgenis.org).
+
+When you log in, you will be able to see at least the following databases:
+
+- **Aggregates**: A database where aggregate data are stored.
+- **Your own database**: The staging area of the resource you are working on.
+Use this to fill out rich metadata and to upload the variable templates once you have filled them out.
+- **catalogue**: The main catalogue, where your data will eventually be transferred to.
+This is also where you can search for target variables to map to.
+- **CatalogueOntologies**: This database contains the ontologies (or look-up lists)
+that you need for filling out some columns in the templates, e.g. format or unit.
+These are centrally managed by us. If you need something added to these look-up lists, [let us know](mailto:support@molgenis.org).
+- **Test databases** (here: testCatalogue, testCohort, testDatasource, testNetwork and testStaging):
+You can see filled out example metadata in these databases.
+
+![MOLGENIS databases](../img/cat_databases.png)
+
+*Figure 1. Databases in the MOLGENIS catalogue.*
+
+## Fill out rich metadata
+
+Open your staging area and open the table **Resources**. The **id** and **name** columns are already filled in.
+Click on the pencil sign next to this entry to start editing your metadata by filling out
+the form. If you are planning on entering source variables,
+make sure to fill in tables **Collection events** and **Subpopulations** as well.
```

Comment:
Entering subpopulations and collection events is not only important when filling out you data dictionaries. The tables also have valuable metadata items for describing your (cohort) dataset in more detail, such as data categories, sample categories.